owa/app/js/components/manage_patient_identifier_sources/view_patient_identifier_sources/remoteIdentifierSource.jsx

```diff
@@ -28,8 +28,7 @@ export default class ViewRemoteIdentifierSource extends Component {
     let data={
       generateIdentifiers: true,
       numberToGenerate: this.state.numberToGenerate,
-      comment: this.state.comment,
-      sourceUuid: this.state.uuid
+      comment: this.state.comment
     }
     apiCall(data, 'post', '/idgen/identifiersource/' + this.state.uuid).then((response) => {
       if( !response["error"] ) {
```

Comment:
Eslint prefers `!response.error` instead of `!response["error"]`

Reply:
Thanks @andela-dbamidele. Fixed.